caffe/indigo/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM rosindustrial/core:indigo
+LABEL maintainer caffe-maint@googlegroups.com
```

Comment:
Can you change the maintainer on both of these to you or I?  Otherwise these look good.